bakerydemo/templates/blog/blog_page.html

```diff
@@ -3,42 +3,35 @@
 
 {% block content %}
 
-    {% image self.image fill-1920x600 as hero_img %}
-        {% include "base/include/header-hero.html" %}
+{% image self.image fill-1920x600 as hero_img %}
+{% include "base/include/header-blog.html" %}
 
-    <div class="container">
-        <div class="row">
-            <div class="col-md-8">
-                {% if page.introduction %}
-                    <p class="intro">{{ page.introduction }}</p>
-                {% endif %}
-
-                <div class="blog-meta">
-                    {% if page.authors %}
-                        <div class="blog-avatars">
-                            {% for author in page.authors %}
-                                <div class="author">{% image author.image fill-50x50-c100 class="blog-avatar" %}
-                                    {{ author.first_name }} {{ author.last_name }}</div>
-                            {% endfor %}
-                        </div>
-                    {% endif %}
-
-                    {% if page.date_published %}
-                        <div class="blog-byline">
-                            {{ page.date_published }}
-                        </div>
-                    {% endif %}
+<div class="container">
+    <div class="row">
+        <div class="col-md-8">
+            <div class="blog__meta">
+                {% if page.authors %}
+                <div class="blog__avatars">
+                    {% for author in page.authors|slice:":1" %}
```

Comment:
Not sure I understand the slice. We don’t display all authors?

Reply:
This was a design decision that Ben Tuckwell suggested, as displaying multiple authors made the section look cluttered. I have removed the slice to show all the information being pulled from Wagtail 👍